python/ray/autoscaler/_private/cluster_dump.py

```diff
@@ -408,7 +408,7 @@ def create_and_get_archive_from_remote_node(
             else ["--no-proccesses-verbose"]
         )
 
-    cmd += ["/bin/bash", "-c", _wrap(collect_cmd, quotes='"')]
+    cmd += ["bash", "-c", _wrap(collect_cmd, quotes='"')]
```

Comment:
this might create security issue to hijacking bash via `PATH`?

maybe the path of bash (or all shell?) should be an configurable option.